dev/intellij-setup.md

```diff
@@ -24,7 +24,7 @@ setups of Druid.
 ## Project SDK
 
 The "SDK", configured for the Druid project, *must* be called `1.8`, to avoid annoying changes of
-[.idea/misc.xml](`.idea/misc.xml`) file, that you shouldn't check into your commits. If you don't have an SDK on your
+`.idea/misc.xml` file, that you shouldn't check into your commits. If you don't have an SDK on your
```

Comment:
These still seem potentially useful to have as links, but they are currently broken, so how about changing to 
```
[`.idea/misc.xml`](/.idea/misc.xml)
```